types/react/jsx-dev-runtime.d.ts

```diff
@@ -1,4 +1,5 @@
 import * as React from "./";
+export { Fragment } from "./";
 
 export namespace JSX {
```

Comment:
I’m not sure why the JSX namespace is duplicated here. It could be:

```js
export { JSX } from "./";
```

Reply:
We can test this in a follow-up. I suspect I duplicated it for module augmentation but I'm not sure we need to support this for `react/jsx-runtime`. Can't see different module augmentations for prod and dev to be useful